style/properties/longhands/position.mako.rs

```diff
@@ -95,7 +95,7 @@ ${helpers.single_keyword(
     // FIXME: Update Servo to support the same Syntax as Gecko.
     ${helpers.single_keyword(
         "justify-content",
-        "flex-start stretch flex-end center space-between space-around",
```

Comment:
Is it possible for Servo to share the version of this for Gecko that's used below now?

Reply:
I discuss this in the PR description. It absolutely is, but that will mean that we're parsing values that we don't actually support (but we already are (`stretch`), and are all over the place when it comes to Flexbox styles).

Reply:
I guess the main question here is do you plan to implement support for these values in the mid-term. I think that if you do plan to add support for those values, it is worth just unifying the implementation now.

Reply:
I think the answer is largely a yes. `normal` is just the default value. The "safe" variants should be pretty trivial to implement (it's just "fallback to `start` on overflow"). Baseline alignment is also very much doable, although tricky to do performantly.

(and on a closer reading of the Gecko implementation the other values I listed aren't relevant - they are there because Gecko is sharing an underling bitflags type between the content alignment and item alignment types but they will never be returned by the parser for these properties).